keyboards/fjlabs/ready100/info.json

```diff
@@ -4,8 +4,11 @@
     "maintainer": "FJLabs",
     "width": 15,
     "height": 5,
+    "layout_aliases": {
+        "LAYOUT_64key": "LAYOUT_64_ansi"
+    },
```

Comment:
This speaks to my inexperience with the configurator, but what is the purpose of this segment?

Reply:
This allows anyone with a keymap that references the old macro name to continue using it without editing. Works when compiling with `make` or `qmk compile` as well.